docs/contribute/getting-started.md

```diff
@@ -30,301 +30,150 @@ page_nav:
+
+## Starting a local SingularityNET Developer Portal
+
+### Download
+
+Get the SingularityNET Developer Portal sources from [Github](https://github.com/singnet/dev-portal), either download an archive or fetch with git clone.
+
+### Start development server
+
+Jekyll comes with a built-in development server that will allow you to preview what the generated site will look like in your browser locally.
+
+You can run this commands inside theme folder:
+
+```
+$ jekyll serve
+# A development server will run at http://localhost:4000/
+# Auto-regeneration: enabled. Use `--no-watch` to disable.
+
+$ jekyll serve --no-watch
+# Same as `jekyll serve` but will not watch for changes.
+
+$ jekyll serve --detach
+# Same as `jekyll serve` but will detach from the current terminal.
+# If you need to kill the server, you can `kill -9 1234` where "1234" is the PID.
+# If you cannot find the PID, then do, `ps aux | grep jekyll` and kill the instance.
+
+```
+
+## Directory structure
+
+This is the basic directory structure which looks like this:
+
+```
+snet/
+├── theme/ # snet theme source files.
+├── _config.yml # Stores Jekyll configuration data.
+├── .eslintrc # ESlint configuration file.
+├── .gitignore # Git related file which specifies intentionally untracked files to ignore.
+├── .gitlab-ci.yml # File used by GitLab Runner to manage your project's jobs.
+├── .htaccess # Configuration file for use on web servers running the Apache Web Server software.
+├── 404.md # Error 404 layout markdown template.
+├── default.md # Default layout markdown template.
+├── favicon.ico # Favicon icon.
+└── index.md # Homepage layout markdown template.
+
+```
+
+## Icons
+
+List of icons you can use in some places such as social list in footer or buttons in homepage layout.
+
+|ICON NAME|YML NAME|PREVIEW|
+|:----------|:---------|:--------|
+|Behance|`behance`| |
+|Bitbucket|`bitbucket`| |
+|Codepen|`codepen`| |
+|Dribbble|`dribbble`| |
+|Dropbox|`dropbox`| |
+|Facebook|`facebook`| |
+|GitHub|`github`| |
+|GitLab|`gitlab`| |
+|Google Plus|`google-plus`| |
+|Gulp|`gulp`| |
+|Instagram|`instagram`| |
+|Kickstarter|`kickstarter`| |
+|LinkedIn|`linkedin`| |
+|Medium|`medium`| |
+|Meetup|`meetup`| |
+|Pocket|`pocket`| |
+|Product Hunt|`producthunt`| |
+|Reddit|`reddit`| |
+|Skype|`skype`| |
+|Slack|`slack`| |
+|Stack Overflow|`stackoverflow`| |
+|Trello|`trello`| |
+|Tumblr|`tumblr`| |
+|Twitter|`twitter`| |
+|YouTube|`youtube`| |
+|Arrow down|`arrow-down`| |
+|Arrow left|`arrow-left`| |
+|Arrow up|`arrow-up`| |
+|Arrow right|`arrow-right`| |
+|Arrow up|`arrow-up`| |
+|Chevron down|`chevron-down`| |
+|Chevron left|`chevron-left`| |
+|Chevron right|`chevron-right`| |
+|Chevron up|`chevron-up`| |
+|Home|`home`| |
+|Maximize|`maximize`| |
+|X (Close)|`x`| |
+
+## Change Log
+
+All notable changes to this project will be documented here. This project adheres to [Semantic Versioning](http://semver.org/).
```

Comment:
I feel like semver and a changelog doesn't really have much use here. These are typically for software, and this is documentation that we are releasing as soon as we've merged PRs.

Reply:
Removed "Changelog" section.